app/views/tags/edit.html.erb

```diff
@@ -95,16 +95,13 @@
           <% if tag_type == 'Fandom' && !@suggested_fandoms.blank? %>
             <dt><%= ts("Suggested Fandoms") %>:</dt>
             <dd>
-            <% @suggested_fandoms[0..19].in_groups(2, false) do |fandom_list| %>
+            <% @suggested_fandoms.in_groups(2, false) do |fandom_list| %>
```

Comment:
Not particularly important since it's exisiting code, but I'm curious: what's the benefit of using `in_groups` here?

Reply:
Honestly not sure there is any, but I didn't want to get into changing the view.